app/adapters/alma/bib.rb

```diff
@@ -13,15 +14,25 @@ class Bib
     # @param records an array of MARC::Record records
     class << self
       def get_bib_record(ids, conn=nil, opts={})
```

Comment:
Let's change the first param to `id` and add yard docs specifying that it should be a string